src/Unsafe/Coerce.purs

```diff
@@ -8,20 +8,84 @@ module Unsafe.Coerce
 -- | (that is, the caller's) responsibility to ensure that the underlying
 -- | representation for both types is the same.
 -- |
--- | One application for this function is to avoid doing work that you know is a
--- | no-op because of newtypes. For example, if you have an `Array (Conj a)` and you
--- | want an `Array (Disj a)`, you could do `Data.Array.map (runConj >>> Disj)`, but
--- | this performs an unnecessary traversal. `unsafeCoerce` accomplishes the same
--- | for free.
--- |
--- | It is highly recommended to define specializations of this function rather than
--- | using it as-is. For example:
--- |
--- | ```purescript
--- | mapConjToDisj :: forall a. Array (Conj a) -> Array (Disj a)
--- | mapConjToDisj = unsafeCoerce
+-- | Note: because this function is extraordinarily flexible, type inference
+-- | can greatly suffer. Whenever you use it, you should add type annotations
+-- | to clarify what the type of it's input value is and the type of it's
```

Comment:
When you're using "its" to indicate possession, there's no apostrophe. You only include the apostrophe in "it's" when you're using "it's" as a contraction of "it is".

Reply:
:smile: I thought that sentence looked weird when I wrote it. Sorry about that.

Reply:
No worries! This is a very common mistake.